### ISSUE TITLE
Silverlight runtime: lastModifiedDate has incorrect value after file is selected using browse dialog

### ISSUE THREAD

**Issue by Imant** [moxiecode/plupload#1256]
Could be reproduced on http://www.plupload.com/examples/runtimes

Just select file and check the value in the javascript console:

$("#silverlight_uploader").pluploadQueue().files[0].lastModifiedDate

lastModifiedDate has value similar to this: "Mon jan 0001 12:00:00 GMT"

**Comment by Imant** [comment]
I know it's because of silverlight doesn't have required permissions to access the file system. But probably it could be fixed somehow in future.

**Comment by aik099** [comment]
If Silverlight doesn't have permission to get file date, then which value you expect to have in the `lastModifiedDate` property. Maybe `null`?

**Comment by Imant** [comment]
No, I expect to see there real last modified date which you can see for each file in the file system. This property should not have another value because of its name.

**Comment by aik099** [comment]
I'm not Silverlight user myself, because I'm on Mac and using regular HTML5 file upload. But from you've described I imagine several cases:

1. The Silverlight can ask user permission and user can deny the request resulting in wrong last modified date
2. The Silverlight can ask user permission and user can approve the request resulting in correct last modified date
3. The Silverlight might not ask user permission at all resulting in wrong last modified date

If you can, then please send a PR with a fix.

**Comment by Imant** [comment]
aik099, the third option.

I am new in Silverlight but I make some googling and the only possible solution I have found is to make silverlight app as trusted in browser: https://msdn.microsoft.com/en-us/library/gg192793(v=vs.95)
